setup.py

```diff
@@ -51,7 +51,7 @@ def main():
         },
         scripts=[],
         zip_safe=False,
-        install_requires=[],
+        install_requires=['requests', 'Pillow'],
```

Comment:
Awesome. Thanks for the tip in the right direction and I'm glad you're excited! :grinning: 

I was thinking about how to keep everything isolated so that the mixin's source doesn't pollute the main source code. That's one of the reasons https://github.com/jhubert/pydocx-s3-images was done as a standalone package.

I am not going to have time to work on this until Wednesday, but I've added you to my fork if you want to take a look / make any commits. I'd love the help. :construction_worker: 

I'm using this in production right now and it's made a huge improvement already. The file sizes I'm getting back are much more reasonable.